lm75bd/lm75bd.c

```diff
@@ -27,8 +30,19 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  uint8_t tmp_reg_pointer = LM75BD_TEMP_REG;
+  i2cSendTo(devAddr, &tmp_reg_pointer, 1);
+  uint8_t buffer[] = {0x00, 0x00};
+  i2cReceiveFrom(devAddr, &buffer[0], 2);
+
+  uint16_t tmp_data =  (((uint16_t) buffer[0]) << 3)  | (((uint16_t) buffer[1])  >> 5);
+  float real_tmp = 0;
+  if (buffer[0] & D10_MASK_BYTE) {
+    real_tmp = - ((tmp_data ^ 0x07FF) + 0x0001) * 0.125;
```

Comment:
Use a macro or add a comment so that it's clear where these numbers are coming from